packages/transition-backend/src/services/odTrip/odTripProvider.ts

```diff
@@ -36,27 +37,27 @@ const extractOdTrip = (
         [key: string]: any;
     },
     options: OdTripOptions,
-    projections: any
+    projection: { srid: number; value: string }
 ): BaseOdTrip => {
     const internalId = line[options.idAttribute];
     const originGeography =
-        options.projection === '4326'
+        projection.srid === constants.geographicCoordinateSystem.srid
             ? turfPoint([parseFloat(line[options.originXAttribute]), parseFloat(line[options.originYAttribute])])
```

Comment:
Not a bit deal either, but this check + conversion could be in a fonction and reused for origin and destination.

Reply:
It's issue #540